modules/euidIdSystem.md

```diff
@@ -0,0 +1,54 @@
+## UID 2.0 User ID Submodule
```

Comment:
this appears to be docs for uid2 not euid

Reply:
Fixed.